controllers/external-accounts.js

```diff
@@ -54,6 +54,7 @@ const syncExternalAccountData = async (req, res) => {
     const [discordUserData, rdsUserData] = await Promise.all([getDiscordMembers(), getDiscordUsers()]);
     const rdsUserDataMap = {};
     const updateUserDataPromises = [];
+    const userUpdatedWithInDiscordFalse = [];
 
```

Comment:
can you please explain me the use of this array, I am not getting it

Reply:
store the user who updated